domain-bootstrap.py

```diff
@@ -417,10 +419,11 @@ def assign_domain(app, domain_profile, svc):
     vars = result_json['variables']
 
     for var in vars:
-        if var['name'] == 'COPILOT_LB_DNS':
+        if var['environment'] == env and var['name'] == 'COPILOT_LB_DNS':
```

Comment:
Might be worth making sure we are pulling this value for/from the correct container.

```suggestion
        if var['environment'] == env and var['name'] == 'COPILOT_LB_DNS' and var['container'] == svc:
```

Reply:
good pick up.  LB can only be attached to a web svc, but when there are multiple services, the copliot command will grab the non web services in its list.

I will add this too.

Reply:
Thank you kindly. Resolved in https://github.com/uktrade/copilot-tools/pull/26/commits/83ef14001f02b1e2b4fd947c652a500a147d3272.